gcp_variant_transforms/libs/bigquery_row_generator.py

```diff
@@ -162,7 +162,9 @@ def _get_base_row_from_variant(self, variant, allow_incompatible_records):
       alt_record = {bigquery_util.ColumnKeyConstants.ALTERNATE_BASES_ALT:
                     alt.alternate_bases}
       for key, data in alt.info.iteritems():
-        alt_record[bigquery_util.get_bigquery_sanitized_field_name(key)] = data
+        if key not in alt._annotation_field_names:
```

Comment:
Please make `alt._annotation_field_names` a property so that you can refer to it without private syntax.